lapce-core/src/syntax/highlight.rs

```diff
@@ -54,6 +54,7 @@ macro_rules! declare_language_highlights {
 declare_language_highlights!(
     Bash: "lang-bash",
     C: "lang-c",
+    CMake: "lang-cmake",
```

Comment:
```suggestion
    Cmake: "lang-cmake",
```

Reply:
The language id as provided in the crate has a capital M

Reply:
Doesn't matter, it's not dependent on external crate